CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# 2.0.0
```

Comment:
Unrelated to the line - I realized post-approval:  The PR shows _This branch is out-of-date with the base branch_.  Is that the intended state?

I didn't diagnose which commit(s) in `master` are _not_ in the PR's branch.

Reply:
@algolucky Resolving - I trust your judgment.